fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/ViewHelpers.java

```diff
@@ -247,6 +248,25 @@ public String getObjectsAsString(final Graph graph,
     }
 
     /**
+     *
+     * @param graph
+     * @param subject
+     * @return
+     */
+    public int getNumChildren(final Graph graph, final Node subject) {
+        LOGGER.trace("Getting number of children: s:{}, g:{}", subject, graph);
+        final Iterator<Node> iterator = listObjects(graph, subject, HAS_CHILD_COUNT.asNode());
```

Comment:
I already told you it did, but I'm using Jena 3. I wouldn't even bother working tickets like this one until that upgrade is in.